code/rounds/InProgressRound.cs

```diff
@@ -128,6 +128,12 @@ private void AssignRoles()
                 {
                     player.Role = new InnocentRole();
                 }
+
+                // send everyone their roles
```

Comment:
Up to you if you want to leave this, but imo this doesn't provide any more info than the comment already does.